components/Alert/index.tsx

```diff
@@ -1,38 +1,51 @@
-import React, { useState } from "react";
-import styles from "./Alert.module.scss";
+import React from 'react';
+import { Button } from '..';
+import styles from './Alert.module.scss';
+
 interface IAlertProps {
   id: string;
   message: string;
   autoClose: boolean;
   type: string;
   onClose: (id: string) => void;
 }
+
+function getBgColor(type: string) {
+  switch (type) {
+    case 'success':
+      return styles.success;
+    case 'error':
```

Comment:
This can be written in very short like
```js
switch(type){
   case "success":
   case "error":
   case "info":
           return styles[type];
   default:
           return styles.info;
}
```

Or we can completely replace this and simply do 
```js 
let bgColor = styles[type];
```
If the correct type is not provided, there shouldn't be any style, it's not coming from any external source like an API, only devs have to do this, type will almost always be correct.